apps/backend/src/temp-scripts/sftp.garms.sh

```diff
@@ -0,0 +1,16 @@
+# ! /bin/bash
```

Comment:
add readme on deets, how to use, export vars, where to find them in our sftp. whom to reach out for pvt keys.